static/specification.html

```diff
@@ -514,210 +271,189 @@ <h2>
-              {{ MonetizationProgressEventInit / amount }} member of
-              {{ MonetizationProgressEventInit }}
-              for more information.
+              {{MonetizationEventInit/amount}} member of
+              {{MonetizationEventInit}} for more information.
             </p>
           </section>
           <section>
-            <h2><dfn>assetCode</dfn> attribute</h2>
+            <h2>
+              <dfn>assetCode</dfn> attribute
+            </h2>
             <p>
               When getting, returns the value it was initialized with. See
-              {{ MonetizationProgressEventInit / assetCode }} member of
-              {{ MonetizationProgressEventInit }} for more information.
+              {{MonetizationEventInit/assetCode}} member of
+              {{MonetizationEventInit}} for more information.
             </p>
           </section>
           <section>
-            <h2><dfn>assetScale</dfn> attribute</h2>
+            <h2>
+              <dfn>assetScale</dfn> attribute
+            </h2>
             <p>
               When getting, returns the value it was initialized with. See
-              {{ MonetizationProgressEventInit / assetScale }} member of
-              {{ MonetizationProgressEventInit }} for more information.
+              {{MonetizationEventInit/assetScale}} member of
+              {{MonetizationEventInit}} for more information.
             </p>
           </section>
           <section>
-            <h2><dfn>receipt</dfn> attribute</h2>
+            <h2>
+              <dfn>receipt</dfn> attribute
+            </h2>
             <p>
               When getting, returns the value it was initialized with. See
-              {{ MonetizationProgressEventInit / receipt }} member of
-              {{ MonetizationProgressEventInit }} for more information.
+              {{MonetizationEventInit/receipt}} member of
+              {{MonetizationEventInit}} for more information.
             </p>
           </section>
-          <section data-dfn-for="MonetizationProgressEventInit">
-            <h3><dfn>MonetizationProgressEventInit</dfn> dictionary</h3>
-            <pre class="idl">
-              dictionary MonetizationProgressEventInit : EventInit {
-                required DOMString amount;
-                required DOMString assetCode;
-                required DOMString assetScale;
+        </section>
+        <section data-dfn-for="MonetizationEventInit">
+          <h3>
+            <dfn>MonetizationEventInit</dfn> dictionary
+          </h3>
+          <p>
+            Dispatched once the first ILP packet with a non-zero amount has
+            been fulfilled by the page's SPSP receiver. MUST be dispatched at
+            least once if payment occurs.
+          </p>
+          <pre class="idl">
+              dictionary MonetizationEventInit : EventInit {
+                required USVString paymentPointer;
+                required DOMString requestId;
+                DOMString amount;
```

Comment:
how come amount isn' required?

Reply:
This was just to cater to the "start" event, when you might not have an amount.  However. the events are created by the (trusted) WM agent, so technically we can assure that `amount` will always be there (same with the rest).

Reply:
If this is rolled into one event, they'll all look the same, and all the optional aren't optional anymore.